lighthouse-core/audits/sized-images.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const i18n = require('./../lib/i18n/i18n.js');
+const URL = require('./../lib/url-shim.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on whether all images had width and height attributes. This descriptive title is shown to users when every image has width and height attributes */
+  title: 'Image elements have `width` and `height` attributes',
+  /** Title of a Lighthouse audit that provides detail on whether all images had width and height attributes. This descriptive title is shown to users when one or more images does not have width and height attributes */
+  failureTitle: 'Image elements do not have `width` and `height` attributes',
+  /** Description of a Lighthouse audit that tells the user why they should include width and height attributes for all images. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
```

Comment:
cc @exterkamp do we need to explain new terms like CLS thrown in here? did we decide not to try to translate the metric acroynms?
```suggestion
  /** Description of a Lighthouse audit that tells the user why they should include width and height attributes for all images. CLS refers to the cumulative layout stability metric and does not need translation. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
```

Reply:
No, we are okay using CLS by itself, it is a known term now. We should avoid comments like "do not translate" because 1.) they can be ignored 😦  2.) we don't know if they should actually not be translated, (looking it up) in some languages CLS is changed slightly.

Reply:
I find it so interesting when metric names change when they're straight from spec events. A bummer that all programming basically happens in english without being i18n'd, but it must add insult to injury to have *even more* alphabet soup to have to match up 😅